examples/gameroom/daemon/src/authorization_handler/mod.rs

```diff
@@ -406,6 +413,48 @@ fn process_admin_event(
     }
 }
 
+fn resubscribe(
```

Comment:
It seems like all of this is duplicated in the `CircuitReady` branch above. Can you rename this method to `subscribe` and call it from the new location as well as in the `CircuitReady` branch?

Reply:
They're close, The WebSocket in `CircuitReady` has an `on_open` callback that creates the xo contract initially. `resubscribe` doesn't have this because it's assumed that the contract already exists when the application authorization handler attempts to resubscribe.

Reply:
Since you're just returning the `WebSocketClient` here anyway, why not call this function from the `CircuitReady` handler, then just add the `on_open` callback to the returned client?

Reply:
They are also very different argument sources in these two cases, so it sill makes sense that they are separate activities.  Let's leave them as-is.